client/components/Editor/plugins/code.ts

```diff
@@ -0,0 +1,73 @@
+import { undo, redo } from 'prosemirror-history';
+import {
+	CodeBlockSettings,
+	codeMirrorBlockPlugin,
+	defaultSettings,
+	languageLoaders,
+	legacyLanguageLoaders,
+} from 'prosemirror-codemirror-block';
+
+import { EditorView } from 'prosemirror-view';
+import { Node } from 'prosemirror-model';
+
+export const myCreateSelect = (
+	settings: CodeBlockSettings,
+	dom: HTMLElement,
+	node: Node,
+	view: EditorView,
+	getPos: (() => number) | boolean,
+) => {
+	if (!settings.languageLoaders) return () => {};
+	const wrapper = document.createElement('div');
+	wrapper.className = 'bp3-html-select codeblock-select-wrapper';
+	const select = document.createElement('select');
+	const carets = document.createElement('span');
+	carets.className = 'bp3-icon bp3-icon-double-caret-vertical';
```

Comment:
`bp3-icon-double-caret-vertical` is having no effect as far as I can see (Firefox). I also think that we should use `caret-down` for consistency with other dropdowns in Blueprint.